usr/sbin/p11sak/p11sak.c

```diff
@@ -1072,6 +1083,7 @@ static CK_RV tok_key_list_init(CK_SESSION_HANDLE session, p11sak_kt kt,
     case kt_SECRET:
     case kt_PUBLIC:
     case kt_PRIVATE:
+    case kt_ALL:
```

Comment:
This case can never happen since above you already return if `kt == kt_ALL`.

Reply:
Done